charts/portworx/templates/portworx-service.yaml

```diff
@@ -19,6 +19,10 @@ spec:
       port: 9019
       targetPort: 9019
     - name: px-sdk
+      protocol: TCP
```

Comment:
let's also remove `type: NodePort`. 

@trierra I just noticed these specs are missing the portworx-api daemonset and the k8s service. Can you add that in a later change?

Reply:
@harsh if I look at: https://install.portworx.com/2.3?mc=false&kbver=1.14.7&b=true&f=true&j=auto&c=px-cluster-a29d12d9-872e-4d54-b87a-b4a74a29e560&stork=true&lh=true&st=k8s , we have:

`type: ClusterIP`, is that what I should change it to?